chirp/drivers/tdh8.py

```diff
@@ -1646,11 +1645,7 @@ def _filter(name):
         _fmv_temp = ''
         for x in range(_fmv_len, -1, -1):
             _fmv_temp += str(_fmv[x])[2:]
-        try:
-            _fmv_block = int(_fmv_temp)
-        except ValueError:
-            LOG.error('Invalid fmv_block %s' % _fmv_temp)
-            _fmv_block = 0
```

Comment:
Why are you reverting this change? It was a recent fix for a reason.

Reply:
I don't remember editing this, I must have fat thumbed something